app/controllers/api/requests_controller.rb

```diff
@@ -1,5 +1,35 @@
 module Api
   class RequestsController < BaseController
+    include Subcollections::RequestTasks
+    include Subcollections::Tasks
+
+    def create_resource(type, _id, data)
+      assert_id_not_specified(data, type)
+
+      request_type = data.delete("request_type")
```

Comment:
`:request_type` is an attribute on `MiqRequest`, so it's probably not safe to use that.